tests/lib/rules/no-unneeded-ternary.js

```diff
@@ -46,14 +47,126 @@ ruleTester.run("no-unneeded-ternary", rule, {
             }]
         },
         {
+            code: "var a = x >= 2 ? true : false;",
+            output: "var a = x >= 2;",
+            errors: [{
+                message: "Unnecessary use of boolean literals in conditional expression.",
+                type: "ConditionalExpression",
+                line: 1,
+                column: 18
+            }]
+        },
+        {
+            code: "var a = x ? true : false;",
+            output: "var a = !!(x);",
```

Comment:
The ternary means the user either explicitly wants `Boolean(x)` or `‼x`, and there's other rules that govern the choice between those. It'd be nice if it didn't add the extra parens, but `no-extra-parens` can handle that too.